sql/hive/src/test/scala/org/apache/spark/sql/hive/test/TestHive.scala

```diff
@@ -649,7 +649,7 @@ private[sql] class TestHiveSessionStateBuilder(
 }
 
 private[hive] object HiveTestJars {
-  private val repository = SQLConf.ADDITIONAL_REMOTE_REPOSITORIES.defaultValueString
+  private val repository = SQLConf.ADDITIONAL_REMOTE_REPOSITORIES.defaultValueString.split(",")(0)
   private val hiveTestJarsDir = Utils.createTempDir()
 
   def getHiveContribJar(version: String = HiveUtils.builtinHiveVersion): File =
```

Comment:
Since you touched this file, could you add a new line after [line 657](https://github.com/apache/spark/pull/26096/files#diff-0818c275c23a3fa4bb8441532bcd046dR657)?

Reply:
Sure!

Reply:
It's done.